build-system/tasks/extension-helpers.js

```diff
@@ -366,16 +367,24 @@ function buildExtension(
       buildExtension(name, version, latestVersion, hasCss, optionsCopy);
     });
   }
-  let promise = Promise.resolve();
+  const promises = [];
   if (hasCss) {
     mkdirSync('build');
     mkdirSync('build/css');
-    promise = buildExtensionCss(path, name, version, optionsCopy);
+    const buildCssPromise = buildExtensionCss(path, name, version, optionsCopy);
     if (options.compileOnlyCss) {
-      return promise;
+      return buildCssPromise;
     }
+
+    promises.push(buildCssPromise);
   }
-  return promise.then(() => {
+
+  // minify and copy vendor configs for amp-analytics component
```

Comment:
Given we are not splitting the config right now. And it also depends on how we want to run the A/B testing. I'd propose that we introduce a flag to compile vendor configs instead of automatically compile with amp-analytics component.

Reply:
Flag is called --compile_vendor_configs